html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>html forms</title>
+</head>
+
+<body>
+  <h1>Coding Form</h1>
+  <form>
+    <div>
+      <label for="email">Email</label>
+      <div>
+        <input type="email" name="email" id="email">
+      </div>
+    </div>
+
+    <div>
+      <label for="why-code">Why is coding cool?</label>
+      <div>
+        <select id="why-code" name="reason">
+          <option value=""> Select an option</option>
```

Comment:
small nitpick but there is an empty space before the text "Select"

Reply:
Ooops, im so sorry about that. Ive removed the leading space in line 22.